### PR TITLE
old pages as stub for expertise added + few style bugs fixes

### PR DESCRIPTION
**Pull request by IlyaKirsanov**

### CODE REVIEW

File: src/components/Expertise/BigDataAnalytics/index.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+
+import { expertiesMainPhoto } from '@/images&video'
+
+export default class BigDataAnalytics extends React.PureComponent {
+  constructor(props) {
+    super(props)
+  }
+  componentDidMount() {
+    this.html = document.getElementsByTagName('html')[0]
```
Comment (sergiylegmelnik) [inline src/components/Expertise/BigDataAnalytics/index.js:13]: Extract this logic to some utility Class

File: src/components/Expertise/ResearchDevelopment/index.js [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+            <div className="container-header" >
+              <Header className="bg-neural" />
+            </div>
+          </section>
+          <div className="application-container" >
+            <div>
+              <span>Research & Development</span>
+              <p>
+                Innovation is crucial to survival. But Innovation is inherently risky.
+                Ciklum R&D experts will cut your risk dramatically by examining the technical feasibility,
```
Comment (sergiylegmelnik) [inline src/components/Expertise/ResearchDevelopment/index.js:36]: Replace all words 'Ciklum' with 'Phobosys'

**Review by sergiylegmelnik — CHANGES_REQUESTED** [review]

File: src/components/Expertise/BigDataAnalytics/index.js (hunk 1 above)
Comment (IlyaKirsanov) [reply src/components/Expertise/BigDataAnalytics/index.js:13]: done

File: src/components/Expertise/ResearchDevelopment/index.js (hunk 2 above)
Comment (IlyaKirsanov) [reply src/components/Expertise/ResearchDevelopment/index.js:36]: done

**Review by sergiylegmelnik — APPROVED** [review]

**Pull request merged**